commands/new_function.go

```diff
@@ -217,37 +223,32 @@ functions:
 
 func printAvailableTemplates(availableTemplates []string) string {
 	var result string
-	sort.Sort(StrSort(availableTemplates))
+	sort.Slice(availableTemplates, func(i, j int) bool {
```

Comment:
Just inlined the sorter? We lost the intent though.. (sort by string), might need comment adding in.